### ISSUE TITLE
addImageBlobHook displays both base64 and server uploaded image

### ISSUE THREAD

**Issue by 2klm** [nhn/tui.editor#1624]
I use addImageBlobHook to upload images to my webserver and render it in the editor. this works fine.
However, the base64 image is still rendered in the editor, I don't want to.

Screenshots : 
https://i.postimg.cc/K3vQrmFx/aaa.png
https://i.postimg.cc/3wCF3v8H/bbbb.png

You can see that the image is displayed twice in the editor; And the html code shows bot the base64 image and the link to the uploaded image

#127 is not fixing this issue.

Here is my code snippet :

hooks: { 
        addImageBlobHook: function (blob, callback) {

            let image = new FormData();

            image.append(0, blob, blob.name);

            fetch('upload.php', {
                method: 'POST',
                body: image
            }).then(function (response) {
                if (response.ok) {
                    response.json().then((value) => { 
                        if (value === 0) { 
                            alert("file upload error");
                        } else { 
                            callback(value, 'uploaded image'); 
                        }
                    });
                } else { 
                    alert("file upload error");
                }
            })
            return false; // I tried with true, and without return at all, same result
        }
    }

**Comment by js87zz** [comment]
@2klm 
The duplication of uploading the image is our bug.
lt will be fixed in #1623.  I'll let you know when it's released!

**Comment by 2klm** [comment]
Removing the default hook and adding it again is a fix.

```
editor.removeHook('addImageBlobHook'); // remove the default hook
editor.addHook('addImageBlobHook', (blob, callback) => { ... });
```

**Comment by js87zz** [comment]
@2klm 
The bug has been fixed and released in [v3.0.1](https://github.com/nhn/tui.editor/releases/tag/editor%403.0.1).
